doc/AARDWOLF-SETUP.MD

```diff
@@ -52,10 +52,16 @@ Then, download the [rustup installer](https://www.rust-lang.org/en-US/install.ht
 
 ### Install Rust Toolchain
 
-Once you have `rustup` installed, make sure you have the `nightly` rust
+Once you have `rustup` installed, make sure you have the `stable` rust
 toolchain installed:
 
```

Comment:
**Suggestion:** The document suggests installing the `nightly` Rust toolchain, but the code seems to be using the `stable` toolchain. Consider updating the document to reflect the actual toolchain used.
```suggestion
Once you have `rustup` installed, make sure you have the `stable` rust
toolchain installed:

    $ rustup toolchain install stable
```